cmd/control/util.go

```diff
@@ -3,13 +3,15 @@ package control
 import (
 	"bufio"
 	"fmt"
+	"os"
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
 )
 
-func yes(in *bufio.Reader, question string) bool {
+func yes(question string) bool {
 	fmt.Printf("%s [y/N]: ", question)
+	in := bufio.NewReader(os.Stdin)
```

Comment:
I could be wrong, but isn't it the nature of the bufio.Reader is that it will buffer data. So if you do

```go
yes("...")
yes("...")
yes("...")
yes("...")
```

You could loose data because it got buffered and discarded.